google/cloud/security/inventory/inventory_loader.py

```diff
@@ -232,6 +233,12 @@ def _build_pipelines(cycle_timestamp, configs, **kwargs):
             bq.BigQueryClient(),
             dao
         ),
+        load_firewall_rules_pipeline.LoadFirewallRulesPipeline(
+            cycle_timestamp,
+            configs,
+            compute.ComputeClient(),
+            kwargs.get('project_dao')
+        ),
```

Comment:
Sorry for asking this of you, but I think I left this a mess. Can you make the line breaks consistent between pipeline args? Also, it seems project_dao and organization_dao are used more than once and should be stuffed into a variable for re-use on L:184